src/components/button/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import style from './button.module.scss';
+
+interface ButtonProps {
```

Comment:
Since we're using TypeScript, we can use a more elegant way for native elements. There are types for them with all possible props. You need just extend it. Then you won't be needed to type all props and pass it to an element manually, you can just spread them.